babeld/files/babeld.init

```diff
@@ -113,34 +113,39 @@ babel_config_cb() {
 		}
 	;;
 	"interface")
+	{
+		local _ignored
+		config_get_bool _ignored "$section" 'ignore' 0
+		[ "$_ignored" == "1" ] && return
+
 		local _ifname
 		config_get _ifname "$section" 'ifname'
+
 		# Try to resolve the logical interface name
-		unset interface
+		local interface
 		network_get_device interface "$_ifname" || interface="$_ifname"
-		option_cb() {
+
+		# Ignore section if ifname is not provided
+		[ -z "$interface" ] &&
+		{
+			logger -s -t babeld "Ignoring interface section which miss ifname"
+			return
```

Comment:
Like above, if interface is empty/not set, option_cb() won't exist. Is this intended?